packages/app-aco/src/components/AdvancedSearch/QueryBuilder/domain/Field.ts

```diff
@@ -0,0 +1,197 @@
+import { CmsModelField } from "@webiny/app-headless-cms-common/types/model";
+
+export type FieldRaw = Pick<
+    CmsModelField,
+    "id" | "type" | "label" | "multipleValues" | "predefinedValues" | "settings"
+>;
+
+export enum TypeDTO {
```

Comment:
I think `TypeDTO` is a misleading name. This is just an `enum`, so we can probably just call it `FieldType`?